apps/dashboard/kinds/v1alpha1/dashboard.cue

```diff
@@ -0,0 +1,19 @@
+package v1alpha1
+
+import (
+	"github.com/grafana/grafana/sdkkinds/dashboard/common"
+)
+
+Dashboard: {
+	schema: {
+		spec:   Spec
+		status: common.DashboardStatus
+	}
+}
+
+// TODO: this outputs nothing.
+// For now, we use unstructured for the spec,
+// but it cannot be produced by the SDK codegen.
+Spec: [string]: _ & {
+	title: string
```

Comment:
v0 and v1 can+should be identical for now

the title property just makes things harder than it needs to be without any real bennifit

Reply:
Just to confirm, do you want me to make https://github.com/grafana/grafana/blob/poc/dashboard-sdk/pkg/apis/dashboard/v1alpha1/dashboard_spec.go and https://github.com/grafana/grafana/blob/poc/dashboard-sdk/pkg/apis/dashboard/v0alpha1/dashboard_spec.go identical?

Because that'd be a breaking change from the PoV of existing code and I'd rather do that in a separate PR.

I'd like to keep the impact of this PR as small as possible, with things working mostly as-is compared to what we have in `main`.

CC @toddtreece @stephaniehingtgen

Reply:
> Because that'd be a breaking change from the PoV of existing code and I'd rather do that in a separate PR.

In https://github.com/grafana/grafana/pull/100845 v0 and v1 (and v2!) all became directly unstructured.Unstructured.  

The explicit title property was just to have *something*, but it only kinda worked and meant getting title from reflection did not work consistently!

Reply:
Oh, I must've missed that change while resolving merge conflicts. This branch has started from way back when we still had title as an explicit field in v1.

I'll update this accordingly then.

Reply:
Done!